src/main/java/de/markherrmann/javauno/service/DrawService.java

```diff
@@ -0,0 +1,92 @@
+package de.markherrmann.javauno.service;
+
+import de.markherrmann.javauno.data.fixed.Card;
+import de.markherrmann.javauno.data.state.component.Game;
+import de.markherrmann.javauno.data.state.component.GameLifecycle;
+import de.markherrmann.javauno.data.state.component.Player;
+import de.markherrmann.javauno.data.state.component.TurnState;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Collections;
+
+@Service
+public class DrawService {
+
+    private final TurnService turnService;
+
+    @Autowired
+    public DrawService(TurnService turnService){
+        this.turnService = turnService;
+    }
+
+    public Card draw(String gameUuid, String playerUuid) throws IllegalArgumentException {
```

Comment:
im throws statement fehlt IllegalStateException
```suggestion
    public Card draw(String gameUuid, String playerUuid) throws IllegalArgumentException, IllegalStateException {
```